Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'sinatra',         '~> 1.4.2'
 gem 'rake',            '~> 0.9.2.2'
 gem 'multi_json'
 
-gem 'sentry-raven',    github: 'getsentry/raven-ruby'
+gem 'sentry-raven',    git: 'https://github.com/getsentry/raven-ruby'
```

Comment:
Should we just switch to the [released](https://rubygems.org/gems/sentry-raven) Sentry gem now that they are post-1.0?